en/docs/learn/design-api/create-api/create-a-rest-api-from-an-openapi-definition.md

```diff
@@ -18,9 +18,27 @@ Follow the instructions below to create a REST API using a Open API definition f
 
     <html><div class="admonition note">
     <p class="admonition-title">Note</p>
-    <p>You need to upload an archive if you have remote references in your API definition. When uploading an OpenAPI archive, the master swagger file should be named as **swagger.yaml**/**swagger.json**. If not, the validation will fail at the point of API creation. Referenced files can be named independently. </p>
+    <p>
+    You need to upload an archive if you have remote references in your API definition. If it is a single swagger file without remote references it can be imported directly, without zipping. 
+    When uploading an OpenAPI archive, the master swagger file should be named as **swagger.yaml**/**swagger.json**. 
+    If not, the validation will fail at the point of API creation. Referenced files can be named independently.
+    </p>
     </div>
     </html>
+    
+    !!! tip
+        Swagger Definitions should be placed under one root folder and zipped.    
+   
+        **Sample Folder Structure**
+    
+        ```
+            -masterFolder
+              ---Swagger.json
+              ---reference.json
```

Comment:
Shall we add a subfolder sample as well? 
ie:
masterFolder
├── reference.yaml
├── schemes
│   └── pet.yaml
└── swagger.yaml

Reply:
Shall we attach a sample here?

Reply:
https://github.com/wso2/docs-apim/pull/1728/commits/99332a4afda62cd5af0e1b492520e36386c96eed

Reply:
is `masterFloder` a required filed name for something or just a sample name? if its a sample shall we rename it to something like `rootDir`

Reply:
It is not mandatory to have a specified name for masterFolder. We did not mention that as a convention to have a name like **masterFolder** . So IMO we don't need to rephrase it.

WDYT ?